CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# 8.6.1
+# 8.7.0
```

Comment:
Could you make this a h2 (`##`) so it's consistent with the other entries?

Reply:
Done! 🎉